### PR TITLE
update rich text editor document

### PR DESCRIPTION
**Pull request by gjulivan**
rich text v 3.2.0

(WC-2479)[https://mendix.atlassian.net/browse/WC-2479]

### CODE REVIEW

**Review by iobuhov — APPROVED** [review]

File: content/en/docs/appstore/widgets/richtext/_index.md [hunk 1]
```diff
@@ -93,7 +93,25 @@ To configure this widget, follow these steps:
         
             {{< figure src="/attachments/appstore/widgets/rich-text/wrap.png" alt="Example" class="no-border" >}}
     
-    * **Location** 
+    * **Location**
+        * **Auto** – let the rich text decide which position is the best location for the toolbar.
```
Comment (takumagott) [inline content/en/docs/appstore/widgets/richtext/_index.md]:
```suggestion
        * **Auto** – automatically determine the best position for the toolbar.
```

File: content/en/docs/appstore/widgets/richtext/_index.md [hunk 2]
```diff
@@ -93,7 +93,25 @@ To configure this widget, follow these steps:
...
+
+        * **Top** – positioned the toolbar at the top of the editor.
```
Comment (takumagott) [inline content/en/docs/appstore/widgets/richtext/_index.md]:
```suggestion
        * **Top** – position the toolbar at the top of the editor.
```

File: content/en/docs/appstore/widgets/richtext/_index.md [hunk 3]
```diff
@@ -93,7 +93,25 @@ To configure this widget, follow these steps:
...
+
+        * **Bottom** – positioned the toolbar at the bottom of the editor.
```
Comment (takumagott) [inline content/en/docs/appstore/widgets/richtext/_index.md]:
```suggestion
        * **Bottom** – position the toolbar at the bottom of the editor.
```

File: content/en/docs/appstore/widgets/richtext/_index.md [hunk 4]
```diff
@@ -93,7 +93,25 @@ To configure this widget, follow these steps:
...
+
+        * **Inline** – positioned the toolbar floating at the current cursor location when editor is in focus. The toolbar will be hidden until the editable content is selected.
```
Comment (takumagott) [inline content/en/docs/appstore/widgets/richtext/_index.md]:
```suggestion
        * **Inline** – position the toolbar floating at the current cursor location when editor is in focus. The toolbar will be hidden until the editable content is selected.
```

File: content/en/docs/appstore/widgets/richtext/_index.md [hunk 5]
```diff
@@ -93,7 +93,25 @@ To configure this widget, follow these steps:
...
+
+        {{% alert color="info" %}}Inline toolbar does not wrap the editor within an iframe, but edits the content in place. Thus, styling that applies to the page, will directly applied to the editor's content.{{% /alert %}}
```
Comment (takumagott) [inline content/en/docs/appstore/widgets/richtext/_index.md]:
```suggestion
        {{% alert color="info" %}}Inline toolbar does not wrap the editor within an iframe, but edits the content in place. Thus, styling that applies to the page will be directly applied to the editor's content.{{% /alert %}}
```

File: content/en/docs/appstore/widgets/richtext/_index.md [hunk 6]
```diff
@@ -93,7 +93,25 @@ To configure this widget, follow these steps:
...
+
+    * **Enable quick toolbar** - If selected, quick toolbar will appear in when content is on focus.
```
Comment (takumagott) [inline content/en/docs/appstore/widgets/richtext/_index.md]:
```suggestion
    * **Enable quick toolbar** - If selected, quick toolbar will appear when content is on focus.
```

File: content/en/docs/appstore/widgets/richtext/_index.md [hunk 7]
```diff
@@ -186,7 +206,13 @@ The full configuration of the menubar groups are as follows:
 
 * **Enable spell checking** – configures to use the browser’s native spell checker. 
 * **Highlight on focus** - adds a blue outline to the editor when that editor is made the input focus
-* **Editor resize** - allows the resize handle to be disabled or to set the resize handler to resize both horizontally and vertically
+* **Additional stylesheet** - allow user to add additional stylesheet that will be included inside editor's iframe.
```
Comment (takumagott) [inline content/en/docs/appstore/widgets/richtext/_index.md]:
```suggestion
* **Additional stylesheet** - allow user to add an additional stylesheet that will be included inside the editor's iframe to apply custom styling.
```

File: content/en/docs/appstore/widgets/richtext/_index.md [hunk 8]
```diff
@@ -186,7 +206,13 @@ The full configuration of the menubar groups are as follows:
...
+    * User can include additional stylesheet file in theme's folder, and add the filename inside the **Additional stylesheet** input.
```
Comment (takumagott) [inline content/en/docs/appstore/widgets/richtext/_index.md]:
```suggestion
    * User can include an additional stylesheet file in the theme folder and add the filename inside the **Additional stylesheet** input. 
```

File: content/en/docs/appstore/widgets/richtext/_index.md [hunk 9]
```diff
@@ -186,7 +206,13 @@ The full configuration of the menubar groups are as follows:
 
 * **Enable spell checking** – configures to use the browser’s native spell checker. 
 * **Highlight on focus** - adds a blue outline to the editor when that editor is made the input focus
-* **Editor resize** - allows the resize handle to be disabled or to set the resize handler to resize both horizontally and vertically
+* **Additional stylesheet** - allow user to add additional stylesheet that will be included inside editor's iframe.
+    * User can include additional stylesheet file in theme's folder, and add the filename inside the **Additional stylesheet** input. 
+    * It is also possible to use built-in stylesheet called "_Default_" and "_Dark_", or alternate between them using the expression editor.
```
Comment (takumagott) [inline content/en/docs/appstore/widgets/richtext/_index.md]:
```suggestion
    * It is also possible to use built-in stylesheets called "_Default_" and "_Dark_", or alternate between them using the expression editor.
```

**Review by takumagott — CHANGES_REQUESTED** [review]
Minor improvement suggestions

**Comment by gjulivan** [comment]
widget release. 
this PR can be merged whenever review done.
@ConnorLand

**Review by ConnorLand — APPROVED** [review]

**Pull request merged**